cmd/tsbs_generate_data/serialize/siridb.go

```diff
@@ -0,0 +1,70 @@
+package serialize
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+
+	qpack "github.com/transceptor-technology/go-qpack"
+)
+
+// SiriDBSerializer writes a Point in a serialized form for TimescaleDB
+type SiriDBSerializer struct{}
+
+// Serialize writes Point p to the given Writer w, so it can be
+// loaded by the TimescaleDB loader. The format is CSV with two lines per Point,
+// with the first row being the tags and the second row being the field values.
+//
+// e.g.,
+// tags,<tag1>,<tag2>,<tag3>,...
+// <measurement>,<timestamp>,<field1>,<field2>,<field3>,...
+func (s *SiriDBSerializer) Serialize(p *Point, w io.Writer) error {
+	line := make([]byte, 4, 1024)
```

Comment:
an explainer comment that the first four bytes will be filled in at the end might be helpful